docs/team/syedomarzoran.md

```diff
@@ -0,0 +1,30 @@
+# Syed Omar Zoran - Project Portfolio Page
+
+## Property Rental Manager - Overview
+Property Rental Manager (PRM) is a desktop application that helps property agents manage, filter and 
+monitor single owner rental units for appropriate tenants.
+
+### Summary of Contributions -
+#### Code contributed - 
+[RepoSense Link](https://nus-cs2113-ay2223s1.github.io/tp-dashboard/?search=zoranabc201&breakdown=true)
+#### Enhancements implemented -
+* Added all the list features for the app -  
+PRM is an app with extensive options for listing the content stored in it - 
+  * Clients - Clients can be listed in various ways. All information present can be listed. A condensed version
+    (called -short) lists the most pertinent information. Individual attributes of clients can also be listed.
+  * Properties - Similar to clients. 
+  * Pairs - Allows listing information about all pairs present in the app. Pairs can be listed either with full 
+  information about their clients and properties, or with condensed information only. This was implemented as a
+  response to a PE-D suggestion and had to be done quickly. Listed items here are not numbered as pairings are not numbered
```

Comment:
I think more elaboration on the complexity of the implementation can be added if needed. I believe you can also add in the tag function if you like to.